Source/Blazorise/Export/Sources/TabularSourceData.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+
+namespace Blazorise.Export;
+
+public class TabularSourceData<TDataType>: IExportableData<TDataType>
+{
+    public List<List<TDataType>> Data { get; set; } = new();
+    public List<string> ColumnNames { get; set; } = new();
```

Comment:
Not sure if `ColumnNames` is a good name for generic exporters? Maybe `KeyNames`, or `FieldNames`?

Reply:
I am also not sure. ColumnNames refer to the table as a structure which has columns and it has some names for them. Csv calls them "Headers".  KeyNames and FieldNames sounds too generic. Which is good if we wanna refer to the output structure (which can be anything).

So - do we wanna refer to the original structure or the output structure?

Reply:
Columns make sense when you speak in terms of tabular data, ot csv/excel files. But since we want to make it as generic as possible, then I think Field is a good approach. We already use Field on DataGrid, and soon on Scheduler. And it also represents the real data that we wish to read or process.

Reply:
But here we **are** talking about `TabularSourceData` class, it doesn't get more generic than that.

Reply:
😅 ignore me.